src/test/java/crawlercommons/robots/SimpleRobotRulesParserTest.java

```diff
@@ -16,6 +16,7 @@
 
 package crawlercommons.robots;
 
+import org.junit.jupiter.api.Assertions;
```

Comment:
Minor: This import seems unnecessary since a static import is already present below (line 37).